### ISSUE TITLE
PLANET-4374 Improve search results to resolve Accessibility and Vertical Spacing

### ISSUE THREAD

**Issue by planet-4** [greenpeace/planet4#89]
Apply spacing vertical and horizontal so that the results are readable and balanced.

Attached are the measurements for the adjustments. The font weights and design layouts follow the existing search results css settings. The knock back on the image is #000000 Alpha 25%.

**Tasks**
- Adjust spacing on the Search results page based on the mockups

**Screenshots**

![Screenshot](https://jira.greenpeace.org/secure/attachment/29843/search_results_adjustments.jpg)
```
Reporter: wmorrisj
Sections: Search
```
Potentially affected repositories: [planet4-master-theme](https://github.com/greenpeace/planet4-master-theme)

**Comment by dpivo** [comment]
**UAT**
Go to https://k8s.p4.greenpeace.org/test-saturn/ 
Search for something

**Comment by magalifatome** [comment]
@dpivo I checked the paddings and the vertical padding between the image and the grey line is not right for XL, L & M screens (see screenshot attached). The vertical padding should be 24px as per in the mockups.
![search results-vertical spacing](https://user-images.githubusercontent.com/66964242/97009784-d3c82200-1544-11eb-90ac-467165761ce7.png)

**Comment by comzeradd** [comment]
Fixed at https://github.com/greenpeace/planet4-master-theme/pull/1236
